geoportailv3.mk

```diff
@@ -16,6 +16,8 @@ PRINT3 = FALSE
 
 DISABLE_BUILD_RULES = test-packages test-packages-ngeo
 
+CONFIG_VARS += ldap_url ldap_bind ldap_passwd ldap_base_dn
```

Comment:
Those variables are missing in the `geoportailv3_vars.yaml` file ...

Not important but I prefer to organized them like it:
```
ldap:
    url: ...
    bind: ...
    passwd: ...
    base_dn: ...
```
than the above line become `CONFIG_VARS += ldap`
and the init:
```
ldap_settings = onfig.get_settings()["ldap"]
config.ldap_setup(
  ldap_settings['url'],
  ldap_settings['bind'],
  ldap_settings['passwd'],
)

config.ldap_set_login_query(
  ldap_settings['base_dn'],
  ...
```
Not a heavy change just more organized settings :-)